apps/beeai-cli/src/beeai_cli/configuration.py

```diff
@@ -31,6 +31,7 @@ class Configuration(pydantic_settings.BaseSettings):
         f"https://github.com/i-am-bee/beeai-platform@v{version()}#path=agent-registry.yaml"
     )
     admin_password: SecretStr | None = None
+    oidc_enabled: bool = False
```

Comment:
we don't use environment variables for this, it's configurable through standardized helm format
```
beeai platform start --set oidc.enabled=true
```